src/Meshes/Meshes.jl

```diff
@@ -164,6 +170,34 @@ function Mesh(; nodes::Vector{N}=Vector{AbstractNode}(),
     Mesh(nodes, elements, faces, node_sets, element_sets, face_sets, extra)
 end
 
+"Return the mesh node coordinates matrix. Each row is a node, each column a coordinate."
+function node_coordinates_matrix(mesh::Mesh{dim,T}) where {dim,T}
+    nodes_coords_matrix = Matrix{Float64}(undef, (num_nodes(mesh), dim))
```

Comment:
i'd expect this to be retrieved from the args, eg. as `Matrix{eltype(T)}(...)`

Reply:
Done